<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite name="Magento Web API Functional Tests" tests="31" assertions="0" errors="30" failures="1" skipped="0" time="11.361107">
    <testsuite name="Magento\Webpos\Api\Cart\CheckoutTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Cart/CheckoutTest.php" tests="6" assertions="0" errors="6" failures="0" skipped="0" time="8.354737">
      <testcase name="testSaveCart" class="Magento\Webpos\Api\Cart\CheckoutTest" classname="Magento.Webpos.Api.Cart.CheckoutTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Cart/CheckoutTest.php" line="291" assertions="0" time="6.052682">
        <error type="Exception">Magento\Webpos\Api\Cart\CheckoutTest::testSaveCart
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 211826934505&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/CurrentSessionId/CurrentSessionIdTest.php:47
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Cart/CheckoutTest.php:51
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Cart/CheckoutTest.php:292
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
      <testcase name="testSaveQuoteData" class="Magento\Webpos\Api\Cart\CheckoutTest" classname="Magento.Webpos.Api.Cart.CheckoutTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Cart/CheckoutTest.php" line="337" assertions="0" time="1.636491">
        <error type="Exception">Magento\Webpos\Api\Cart\CheckoutTest::testSaveQuoteData
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1327347195430&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/CurrentSessionId/CurrentSessionIdTest.php:47
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Cart/CheckoutTest.php:339
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
      <testcase name="testSaveShippingMethod" class="Magento\Webpos\Api\Cart\CheckoutTest" classname="Magento.Webpos.Api.Cart.CheckoutTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Cart/CheckoutTest.php" line="410" assertions="0" time="0.154631">
        <error type="Exception">Magento\Webpos\Api\Cart\CheckoutTest::testSaveShippingMethod
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1190193065197&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/CurrentSessionId/CurrentSessionIdTest.php:47
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Cart/CheckoutTest.php:412
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
      <testcase name="testPlaceOrder" class="Magento\Webpos\Api\Cart\CheckoutTest" classname="Magento.Webpos.Api.Cart.CheckoutTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Cart/CheckoutTest.php" line="468" assertions="0" time="0.145195">
        <error type="Exception">Magento\Webpos\Api\Cart\CheckoutTest::testPlaceOrder
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1118568130582&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/CurrentSessionId/CurrentSessionIdTest.php:47
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Cart/CheckoutTest.php:470
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
      <testcase name="testApplyGiftcard" class="Magento\Webpos\Api\Cart\CheckoutTest" classname="Magento.Webpos.Api.Cart.CheckoutTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Cart/CheckoutTest.php" line="625" assertions="0" time="0.216457">
        <error type="Exception">Magento\Webpos\Api\Cart\CheckoutTest::testApplyGiftcard
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1508695435514&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/CurrentSessionId/CurrentSessionIdTest.php:47
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Cart/CheckoutTest.php:627
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
      <testcase name="testSpendPoint" class="Magento\Webpos\Api\Cart\CheckoutTest" classname="Magento.Webpos.Api.Cart.CheckoutTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Cart/CheckoutTest.php" line="719" assertions="0" time="0.149281">
        <error type="Exception">Magento\Webpos\Api\Cart\CheckoutTest::testSpendPoint
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1338014740058&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/CurrentSessionId/CurrentSessionIdTest.php:47
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Cart/CheckoutTest.php:721
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Catalog\CategoryRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Catalog/CategoryRepositoryTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.141753">
      <testcase name="testGetList" class="Magento\Webpos\Api\Catalog\CategoryRepositoryTest" classname="Magento.Webpos.Api.Catalog.CategoryRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Catalog/CategoryRepositoryTest.php" line="35" assertions="0" time="0.141753">
        <error type="Exception">Magento\Webpos\Api\Catalog\CategoryRepositoryTest::testGetList
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1170381914105&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:51
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:95
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Catalog/CategoryRepositoryTest.php:51
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Catalog\ProductRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Catalog/ProductRepositoryTest.php" tests="2" assertions="0" errors="2" failures="0" skipped="0" time="0.295306">
      <testcase name="testGetList" class="Magento\Webpos\Api\Catalog\ProductRepositoryTest" classname="Magento.Webpos.Api.Catalog.ProductRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Catalog/ProductRepositoryTest.php" line="36" assertions="0" time="0.147399">
        <error type="Exception">Magento\Webpos\Api\Catalog\ProductRepositoryTest::testGetList
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 448036826588&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:51
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:95
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Catalog/ProductRepositoryTest.php:60
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
      <testcase name="testGetOptions" class="Magento\Webpos\Api\Catalog\ProductRepositoryTest" classname="Magento.Webpos.Api.Catalog.ProductRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Catalog/ProductRepositoryTest.php" line="87" assertions="0" time="0.147907">
        <error type="Exception">Magento\Webpos\Api\Catalog\ProductRepositoryTest::testGetOptions
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 761967392272&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:51
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:95
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Catalog/ProductRepositoryTest.php:96
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Catalog\SwatchRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Catalog/SwatchRepositoryTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.142868">
      <testcase name="testGetList" class="Magento\Webpos\Api\Catalog\SwatchRepositoryTest" classname="Magento.Webpos.Api.Catalog.SwatchRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Catalog/SwatchRepositoryTest.php" line="35" assertions="0" time="0.142868">
        <error type="Exception">Magento\Webpos\Api\Catalog\SwatchRepositoryTest::testGetList
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1357825893309&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:51
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:95
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Catalog/SwatchRepositoryTest.php:51
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Checkout\CartTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/CartTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.150367">
      <testcase name="testSendEmail" class="Magento\Webpos\Api\Checkout\CartTest" classname="Magento.Webpos.Api.Checkout.CartTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/CartTest.php" line="24" assertions="0" time="0.150367">
        <error type="Exception">Magento\Webpos\Api\Checkout\CartTest::testSendEmail
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 812257240502&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/CartTest.php:38
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Checkout\TaxClassTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/TaxClassTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.143561">
      <testcase name="testGetList" class="Magento\Webpos\Api\Checkout\TaxClassTest" classname="Magento.Webpos.Api.Checkout.TaxClassTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/TaxClassTest.php" line="24" assertions="0" time="0.143561">
        <error type="Exception">Magento\Webpos\Api\Checkout\TaxClassTest::testGetList
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1170381915260&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:51
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:95
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Checkout/TaxClassTest.php:40
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Config\ConfigRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Config/ConfigRepositoryTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.141222">
      <testcase name="testGetList" class="Magento\Webpos\Api\Config\ConfigRepositoryTest" classname="Magento.Webpos.Api.Config.ConfigRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Config/ConfigRepositoryTest.php" line="24" assertions="0" time="0.141222">
        <error type="Exception">Magento\Webpos\Api\Config\ConfigRepositoryTest::testGetList
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 736060501547&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:51
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:95
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Config/ConfigRepositoryTest.php:33
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\CurrentSessionId\CurrentSessionIdTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/CurrentSessionId/CurrentSessionIdTest.php" tests="1" assertions="0" errors="0" failures="1" skipped="0" time="0.000714">
      <testcase name="Warning" assertions="0" time="0.000714">
        <warning type="PHPUnit\Framework\Warning">Warning
No tests found in class "Magento\Webpos\Api\CurrentSessionId\CurrentSessionIdTest".

/var/www/vendor/phpunit/phpunit/phpunit:53
</warning>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Customer\CustomerRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Customer/CustomerRepositoryTest.php" tests="2" assertions="0" errors="2" failures="0" skipped="0" time="0.300395">
      <testcase name="testGetList" class="Magento\Webpos\Api\Customer\CustomerRepositoryTest" classname="Magento.Webpos.Api.Customer.CustomerRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Customer/CustomerRepositoryTest.php" line="35" assertions="0" time="0.152180">
        <error type="Exception">Magento\Webpos\Api\Customer\CustomerRepositoryTest::testGetList
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1208480285551&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:51
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:95
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Customer/CustomerRepositoryTest.php:57
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
      <testcase name="testSave" class="Magento\Webpos\Api\Customer\CustomerRepositoryTest" classname="Magento.Webpos.Api.Customer.CustomerRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Customer/CustomerRepositoryTest.php" line="83" assertions="0" time="0.148215">
        <error type="Exception">Magento\Webpos\Api\Customer\CustomerRepositoryTest::testSave
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 629385066848&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Customer/CustomerRepositoryTest.php:103
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Inventory\StockItemRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Inventory/StockItemRepositoryTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.146837">
      <testcase name="testGetStockItems" class="Magento\Webpos\Api\Inventory\StockItemRepositoryTest" classname="Magento.Webpos.Api.Inventory.StockItemRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Inventory/StockItemRepositoryTest.php" line="24" assertions="0" time="0.146837">
        <error type="Exception">Magento\Webpos\Api\Inventory\StockItemRepositoryTest::testGetStockItems
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1022560242017&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:51
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:95
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Inventory/StockItemRepositoryTest.php:50
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Payment\PaymentRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Payment/PaymentRepositoryTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.142637">
      <testcase name="testGetList" class="Magento\Webpos\Api\Payment\PaymentRepositoryTest" classname="Magento.Webpos.Api.Payment.PaymentRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Payment/PaymentRepositoryTest.php" line="35" assertions="0" time="0.142637">
        <error type="Exception">Magento\Webpos\Api\Payment\PaymentRepositoryTest::testGetList
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1315155721382&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:51
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:95
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Payment/PaymentRepositoryTest.php:44
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Pos\PosRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Pos/PosRepositoryTest.php" tests="2" assertions="0" errors="2" failures="0" skipped="0" time="0.362623">
      <testcase name="testGetList" class="Magento\Webpos\Api\Pos\PosRepositoryTest" classname="Magento.Webpos.Api.Pos.PosRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Pos/PosRepositoryTest.php" line="44" assertions="0" time="0.213413">
        <error type="Exception">Magento\Webpos\Api\Pos\PosRepositoryTest::testGetList
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1370017374090&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:51
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:95
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Pos/PosRepositoryTest.php:75
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
      <testcase name="testAssignStaff" class="Magento\Webpos\Api\Pos\PosRepositoryTest" classname="Magento.Webpos.Api.Pos.PosRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Pos/PosRepositoryTest.php" line="112" assertions="0" time="0.149210">
        <error type="Exception">Magento\Webpos\Api\Pos\PosRepositoryTest::testAssignStaff
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 781778546221&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/CurrentSessionId/CurrentSessionIdTest.php:47
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Pos/PosRepositoryTest.php:123
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Sales\CreditmemoRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Sales/CreditmemoRepositoryTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.154162">
      <testcase name="testSaveCreditmemo" class="Magento\Webpos\Api\Sales\CreditmemoRepositoryTest" classname="Magento.Webpos.Api.Sales.CreditmemoRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Sales/CreditmemoRepositoryTest.php" line="35" assertions="0" time="0.154162">
        <error type="Exception">Magento\Webpos\Api\Sales\CreditmemoRepositoryTest::testSaveCreditmemo
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1008844829849&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Sales/CreditmemoRepositoryTest.php:59
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Sales\InvoiceRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Sales/InvoiceRepositoryTest.php" tests="2" assertions="0" errors="2" failures="0" skipped="0" time="0.296856">
      <testcase name="testUpdateQty" class="Magento\Webpos\Api\Sales\InvoiceRepositoryTest" classname="Magento.Webpos.Api.Sales.InvoiceRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Sales/InvoiceRepositoryTest.php" line="36" assertions="0" time="0.147935">
        <error type="Exception">Magento\Webpos\Api\Sales\InvoiceRepositoryTest::testUpdateQty
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1014940569218&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Sales/InvoiceRepositoryTest.php:66
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
      <testcase name="testSaveInvoice" class="Magento\Webpos\Api\Sales\InvoiceRepositoryTest" classname="Magento.Webpos.Api.Sales.InvoiceRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Sales/InvoiceRepositoryTest.php" line="79" assertions="0" time="0.148921">
        <error type="Exception">Magento\Webpos\Api\Sales\InvoiceRepositoryTest::testSaveInvoice
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 246877435793&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/CurrentSessionId/CurrentSessionIdTest.php:47
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Sales/InvoiceRepositoryTest.php:80
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Sales\OrderRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Sales/OrderRepositoryTest.php" tests="2" assertions="0" errors="2" failures="0" skipped="0" time="0.309603">
      <testcase name="testGetList" class="Magento\Webpos\Api\Sales\OrderRepositoryTest" classname="Magento.Webpos.Api.Sales.OrderRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Sales/OrderRepositoryTest.php" line="35" assertions="0" time="0.155460">
        <error type="Exception">Magento\Webpos\Api\Sales\OrderRepositoryTest::testGetList
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 434321414918&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:51
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:95
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Sales/OrderRepositoryTest.php:57
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
      <testcase name="testAddComment" class="Magento\Webpos\Api\Sales\OrderRepositoryTest" classname="Magento.Webpos.Api.Sales.OrderRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Sales/OrderRepositoryTest.php" line="142" assertions="0" time="0.154143">
        <error type="Exception">Magento\Webpos\Api\Sales\OrderRepositoryTest::testAddComment
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1060658613440&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Sales/OrderRepositoryTest.php:161
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Shift\CashTransactionRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Shift/CashTransactionRepositoryTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.138739">
      <testcase name="testSave" class="Magento\Webpos\Api\Shift\CashTransactionRepositoryTest" classname="Magento.Webpos.Api.Shift.CashTransactionRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Shift/CashTransactionRepositoryTest.php" line="41" assertions="0" time="0.138739">
        <error type="Exception">Magento\Webpos\Api\Shift\CashTransactionRepositoryTest::testSave
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1264865875485&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Shift/ShiftRepositoryTest.php:309
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Shift/CashTransactionRepositoryTest.php:43
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Shift\ShiftRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Shift/ShiftRepositoryTest.php" tests="4" assertions="0" errors="4" failures="0" skipped="0" time="0.001271">
      <testcase name="testGetList" class="Magento\Webpos\Api\Shift\ShiftRepositoryTest" classname="Magento.Webpos.Api.Shift.ShiftRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Shift/ShiftRepositoryTest.php" line="49" assertions="0" time="0.000724">
        <error type="Error">Magento\Webpos\Api\Shift\ShiftRepositoryTest::testGetList
Error: Undefined class constant 'SCOPE_DEFAULT'

/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Shift/ShiftRepositoryTest.php:41
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
      <testcase name="testSave" class="Magento\Webpos\Api\Shift\ShiftRepositoryTest" classname="Magento.Webpos.Api.Shift.ShiftRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Shift/ShiftRepositoryTest.php" line="160" assertions="0" time="0.000171">
        <error type="Error">Magento\Webpos\Api\Shift\ShiftRepositoryTest::testSave
Error: Undefined class constant 'SCOPE_DEFAULT'

/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Shift/ShiftRepositoryTest.php:41
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
      <testcase name="testValidateSession" class="Magento\Webpos\Api\Shift\ShiftRepositoryTest" classname="Magento.Webpos.Api.Shift.ShiftRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Shift/ShiftRepositoryTest.php" line="192" assertions="0" time="0.000168">
        <error type="Error">Magento\Webpos\Api\Shift\ShiftRepositoryTest::testValidateSession
Error: Undefined class constant 'SCOPE_DEFAULT'

/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Shift/ShiftRepositoryTest.php:41
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
      <testcase name="testOpenSession" class="Magento\Webpos\Api\Shift\ShiftRepositoryTest" classname="Magento.Webpos.Api.Shift.ShiftRepositoryTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Shift/ShiftRepositoryTest.php" line="322" assertions="0" time="0.000208">
        <error type="Error">Magento\Webpos\Api\Shift\ShiftRepositoryTest::testOpenSession
Error: Undefined class constant 'SCOPE_DEFAULT'

/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Shift/ShiftRepositoryTest.php:41
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
    </testsuite>
    <testsuite name="Magento\Webpos\Api\Staff\StaffManagementTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/StaffManagementTest.php" tests="1" assertions="0" errors="1" failures="0" skipped="0" time="0.137456">
      <testcase name="testStaffLogin" class="Magento\Webpos\Api\Staff\StaffManagementTest" classname="Magento.Webpos.Api.Staff.StaffManagementTest" file="/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/StaffManagementTest.php" line="31" assertions="0" time="0.137456">
        <error type="Exception">Magento\Webpos\Api\Staff\StaffManagementTest::testStaffLogin
Exception: &lt;!doctype html&gt;
&lt;html xmlns="http://www.w3.org/1999/xhtml" &gt;
&lt;head&gt;
    &lt;title&gt;There has been an error processing your request&lt;/title&gt;
    &lt;base href="http://magento2.travis/pub/errors/default/" /&gt;
    &lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
    &lt;meta name="robots" content="*"/&gt;
    &lt;link rel="stylesheet" href="css/styles.css" type="text/css" /&gt;
    &lt;link rel="icon" href="images/favicon.ico" type="image/x-icon" /&gt;
    &lt;link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon" /&gt;
&lt;/head&gt;
&lt;body&gt;
    &lt;main class="page-main"&gt;
        
&lt;h1&gt;There has been an error processing your request&lt;/h1&gt;

&lt;div class="trace-container"&gt;
    &lt;div class="trace"&gt;
        &lt;pre&gt;Can't create directory /var/www/generated/code/Magento/Webapi/Controller/Rest/Router/. in [Magento\Webapi\Controller\Rest\Router\Proxy]&lt;/pre&gt;
    &lt;/div&gt;
&lt;/div&gt;
    &lt;p&gt;Error log record number: 1094185179547&lt;/p&gt;
    &lt;/main&gt;
&lt;/body&gt;
&lt;/html&gt;


/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:216
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:134
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest/CurlClient.php:66
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/Webapi/Adapter/Rest.php:98
/var/www/dev/tests/api-functional/framework/Magento/TestFramework/TestCase/WebapiAbstract.php:175
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/CurrentSessionId/CurrentSessionIdTest.php:47
/var/www/dev/tests/api-functional/testsuite/Magento/Webpos/Api/Staff/StaffManagementTest.php:33
/var/www/vendor/phpunit/phpunit/phpunit:53
</error>
      </testcase>
    </testsuite>
  </testsuite>
</testsuites>
